code/game/atoms.dm

```diff
@@ -393,17 +392,15 @@ its easier to just keep the beam vertical.
 
 	if(cur_x && cur_y)
 		return list("x"=cur_x,"y"=cur_y)
-	else
-		return 0
+	return 0
 
 /atom/proc/checkpass(passflag)
 	return pass_flags&passflag
 
 /atom/proc/isinspace()
 	if(istype(get_turf(src), /turf/space))
 		return 1
-	else
-		return 0
+	return 0
```

Comment:
All the procs modified in this file could just return their conditional instead of explicitly returning true or false.

Reply:
See, I was told not to change that. I can fix that right now though!